src/build-babel-preset-env.js

```diff
@@ -0,0 +1,218 @@
+import pluginList from "babel-preset-env/data/plugins.json";
```

Comment:
Is there an advantage to having this in a separate file? What about just putting this directly into `babel-preset-env`?